tests/unit/s2n_fragmentation_coalescing_test.c

```diff
@@ -435,7 +435,7 @@ int main(int argc, char **argv)
         fragmented_message(p[1]);
         EXPECT_SUCCESS(s2n_config_free(config));
         EXPECT_SUCCESS(s2n_connection_free(conn));
-        _exit(0);
+        exit(0);
```

Comment:
What exactly does switching this function have to do with memory?

Reply:
I also call this out in the description, but basically _exit doesnt call the `atexit` routine and doesnt do a full cleanup in the child process.